homeassistant/components/mobile_app/notify.py

```diff
@@ -89,13 +89,12 @@ def targets(self):
         targets = kwargs.get(ATTR_TARGET)
 
         if not targets:
-            targets = push_registrations(self.hass)
+            targets = push_registrations(self.hass).values()
```

Comment:
Just FYI, the return type of this is an iterator - it will work here because it's only used in the for loop but it might cause problems.